.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+on: pull_request
+permissions: {}
+jobs:
+  lint:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
```

Comment:
Is it possible to use tags instead of commit hashes?

Reply:
It's possible, but AppSec recommends to use hashes.